pkg/replication/replication.go

```diff
@@ -988,10 +988,10 @@ func (q ReplQueueStats) QStats() (r ReplQStats) {
 
 // MetricsV2 represents replication metrics for a bucket.
 type MetricsV2 struct {
-	Uptime       int64          `json:"uptime"`
-	CurrentStats Metrics        `json:"currStats"`
-	QueueStats   ReplQueueStats `json:"queueStats"`
-	DowntimeInfo DowntimeInfo   `json:"downtimeInfo"`
+	Uptime       int64                   `json:"uptime"`
+	CurrentStats Metrics                 `json:"currStats"`
+	QueueStats   ReplQueueStats          `json:"queueStats"`
+	DowntimeInfo map[string]DowntimeInfo `json:"downtimeInfo"`
```

Comment:
We cannot break APIs and Structs in this manner, add it as a new entry - you are even retaining the JSON key which will cause all sorts of problems.

Reply:
+1. Rather have a new field say `DowntimeInfoV2`

Reply:
this field was recently added and new.  Not yet used by any APIs. This was added in https://github.com/minio/minio-go/pull/2071 

I had to change this to address per target downtime info in my WIP - https://github.com/miniohq/ec/pull/37

Reply:
consider this as a continuation of https://github.com/minio/minio-go/pull/2071

Reply:
Oh ok, then it should be fine

Reply:
Unfortunately we have made a release already, please use a different filed https://github.com/minio/minio-go/releases/tag/v7.0.88 @Praveenrajmani

Reply:
the server changes which sets this field was never merged or released @harshavardhana do we still need a new field?

Reply:
should be fine to leave this as is

Reply:
Okay

Reply:
Please make sure to avoid these in the future.